From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 18 May 2016 23:09:54 +0200
Subject: Fix typo: Recieve -> Received

---
 exec/totemsrp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/exec/totemsrp.c b/exec/totemsrp.c
index 3aa61cc..9c8edc4 100644
--- a/exec/totemsrp.c
+++ b/exec/totemsrp.c
@@ -4223,7 +4223,7 @@ static void memb_join_process (
 		} else {
 			if (memb_join->header.nodeid == LEAVE_DUMMY_NODEID) {
 				log_printf (instance->totemsrp_log_level_debug,
-		    		"Recieve LEAVE message from %u", memb_join->failed_list_entries > 0 ? failed_list[memb_join->failed_list_entries - 1 ].addr[0].nodeid : LEAVE_DUMMY_NODEID);
+		    		"Received LEAVE message from %u", memb_join->failed_list_entries > 0 ? failed_list[memb_join->failed_list_entries - 1 ].addr[0].nodeid : LEAVE_DUMMY_NODEID);
 				if (memb_join->failed_list_entries > 0) {
 					my_leave_memb_set(instance, failed_list[memb_join->failed_list_entries - 1 ].addr[0].nodeid);
 				}
